### DISCUSSION TITLE
[FR] Multiple toolbars or at least one "Global Hover Button"?

### DISCUSSION THREAD

**Discussion by Moyf** [chrisgurney/obsidian-note-toolbar#111] [Ideas]
Is there a way to support displaying multiple toolbars on a page? 

With the introduction of the floating button feature on the PC platform, I created a "Universal Function" button— 
![image](https://github.com/user-attachments/assets/0b008cf5-985a-4085-aada-813b81f4b972)

I hope it would display in all of my notes, but once it's on a page that already has a toolbar, this button cannot be displayed. I can only use MenuItem to attach it to the end of the toolbar.
![image](https://github.com/user-attachments/assets/19e13d10-e73e-4fba-8783-52932d028064)

---

I know supporting multiple toolbar would make it more complex and not very necessary (Since we now have the MenuItem), but at least ONE GLOBAL TOOLBAR (Float button) might be more convinient? 🥺

**Comment by chrisgurney** [comment]
@Moyf Interesting thought! I'll have a think on this one.

As an aside, this image looks like a bug, possibly? The menu shouldn't be that far from the floating button. Can you share more about how the button's positioned and configured, and maybe a video? (But it looks like there might be another button behind the menu?) 

If you could open a separate issue you think it needs it, that would be helpful!
![image](https://github.com/user-attachments/assets/8b83798c-702d-4ec5-abc2-04ea4489af22)

**Reply by Moyf** [reply]
> @Moyf Interesting thought! I'll have a think on this one.
> 
> As an aside, this image looks like a bug, possibly? The menu shouldn't be that far from the floating button. Can you share more about how the button's positioned and configured, and maybe a video? (But it looks like there might be another button behind the menu?)
> 
> If you could open a separate issue you think it needs it, that would be helpful! ![image](https://private-user-images.githubusercontent.com/17337703/352716893-8b83798c-702d-4ec5-abc2-04ea4489af22.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIxNDM2MDUsIm5iZiI6MTcyMjE0MzMwNSwicGF0aCI6Ii8xNzMzNzcwMy8zNTI3MTY4OTMtOGI4Mzc5OGMtNzAyZC00ZWM1LWFiYzItMDRlYTQ0ODlhZjIyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzI4VDA1MDgyNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRjNzIxZDhkODEyOTJjNmQyNmYyNjRhNDk2MjY5NzcwODA2ZTdiNmMzMjQxMjUxYjhhNmYwYWM1MjA1MmNlZDgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.p_MlGKO_QCPKq-lVXvNsFQ6A7h5PttKdXH_aLZRYtmY)

Hahaha no no, this image is combined from 2 images, just to show "how I use it" lol
Sorry for the confusin 😹

**Comment by chrisgurney** [comment]
Another vote for multiple toolbars from the feedback form:

"Being able to add multiple toolbars to a document would be nice too (for example having a toolbar at the top AND the floating button at the right)"

**Comment by Moyf** [comment]
I found  new plugin: [Advanced Debug Mode](https://github.com/mnaoumov/obsidian-advanced-debug-mode). On mobile it adds a button you can position freely wherever you want, just for your information  :D 
<img width="317" alt="image" src="https://github.com/user-attachments/assets/cc7288fa-2122-42e8-a44b-be024c83cde2" />

(Personally I don't use mobile too much, haha)

**Comment by chrisgurney** [comment]
@Moyf Nice find. 

Looks like this plugin is a wrapper for [Eruda](https://github.com/liriliri/eruda) which has this component, which _looks_ like it's the one:
https://github.com/liriliri/eruda/blob/master/src/EntryBtn/EntryBtn.js

**Comment by n-konig** [comment]
+1 for this feature! Would love to be able to use top toolbar mainly for navigation and information and a floating toolbar for quick common commands. (Thx for the SUPERB plugin!)
